src/zzz_od/application/inventory_scan/InventoryDataProcessor.py

```
@@ -0,0 +1,486 @@
+        'B': 0.33
+    }
+    
+    # 配置每个部位的主词条可选池
+    SLOT_MAIN_POOLS = {
+            1: ['小生命'],           # 1号位固定主词条
+            2: ['小攻击'],          # 2号位固定主词条
+            3: ['小防御'],         # 3号位固定主词条
+            4: ['攻击力', '生命值', '防御力', '暴击率', '暴击伤害', '异常精通'],  # 4号位可选主词条
+            5: ['攻击力', '生命值', '防御力', '穿透率', '火属性伤害加成', '冰属性伤害加成', '电属性伤害加成', '以太属性伤害加成', '物理属性伤害加成'],  # 5号位可选主词条
+            6: ['攻击力', '生命值', '防御力', '异常掌控', '冲击力', '能量自动回复'],  # 6号位可选主词条
+        }
```

Ruff (0.15.7)

[warning] Mutable default value for class attribute

(RUF012)